frontend/src/App.vue

```diff
@@ -241,8 +241,15 @@ export default {
           }
         }
       } catch (error) {
-        const message = `'${question.name}' question update of generator ${this.generatorName} has failed: ${_.get(error, "message", error)}`;
-        await this.rpc.invoke("logMessage", [message]);
+        let errorInfo = "No error info";
+        if (_.isString(error)) {
+          errorInfo = error;
+        } else {
+          errorInfo = _.get(error, "stack", errorInfo);
+        }
```

Comment:
could be that there's no stuck, but error.message exist.

Reply:
fixed